cli/src/commands/shared_recovery/delete.rs

```diff
@@ -0,0 +1,30 @@
+use crate::utils::*;
+
+crate::clap_parser_with_shared_opts_builder!(
+    #[with = config_dir, device, password_stdin]
+    pub struct Args {
+    }
+);
+
+pub async fn main(args: Args) -> anyhow::Result<()> {
+    let Args {
+        password_stdin,
+        device,
+        config_dir,
+    } = args;
+
+    let client = load_client(&config_dir, device, password_stdin).await?;
+
+    {
+        let _spinner = start_spinner("Poll server for new certificates".into());
+        client.poll_server_for_new_certificates().await?;
+    }
+
+    client.delete_shamir_recovery().await?;
```

Comment:
A client can only have a single shared recovery ?

Reply:
yes, only a single shamir recovery is allowed, you need to delete the previous one first before registering a new one